lint.bat

```diff
@@ -2,6 +2,6 @@
 
 dotnet format --verbosity=d --verify-no-changes --severity=warn .\ChainSafe.Gaming.sln --exclude .\submodules
 
-pushd src/UnitySampleProject
+pushd .\src\UnitySampleProject
 dotnet format --verbosity=d --verify-no-changes --severity=warn .\UnitySampleProject.sln
```

Comment:
```suggestion
dotnet format --verbosity=d --severity=warn .\UnitySampleProject.sln
```

Reply:
in order for a script to actually lint the code this argument needs to be removed from both commands